packages/client/src/client-utils.ts

```diff
@@ -73,13 +79,19 @@ export function paramsByLocation(definition: AnyInputDef, input: Record<string,
     const param = definition[propertyName];
     if (param.location === 'body') {
       body = param instanceof BodyBinaryParamDef ? (rawValue as Blob) : paramStringValue(rawValue);
+      bodyType =
```

Comment:
Would it make sense to add a test for this function, similar to the other functions in client-utils? It would help ensure we get the correct bodyType based on the parameter.